apps/mobile-wallet/src/components/layout/ScrollScreen.tsx

```diff
@@ -43,6 +44,7 @@ export interface ScrollScreenProps extends ScrollScreenBaseProps, ScrollViewProp
   scrollViewRef?: RefObject<ScrollView>
   verticalGap?: number | boolean
   contentPaddingTop?: number | boolean
+  screenIntroPaddingBottom?: boolean
```

Comment:
Is there a case where we have `screenIntro` but we don't want padding bottom? If there is maybe it's worth it to refactor those cases so that the UI is consistent. It's a weird prop to have imo.

Reply:
You're right, I've tried multiple approaches but I actually can do easily without it with the current solution I believe.